app/api/campaigns/[campaignId]/route.ts

```
@@ -1,22 +1,34 @@
 import { getCampaign } from '@/lib/api/campaigns';
 import { db } from '@/server/db';
-import { checkAuth } from '@/lib/api/auth';
+import { checkAuth, isAdmin } from '@/lib/api/auth';
 import {
   ApiParameterError,
   ApiNotFoundError,
   ApiIntegrityError,
+  ApiAuthNotAllowed,
```

ESLint

[error] 'ApiAuthNotAllowed' is defined but never used.

(@typescript-eslint/no-unused-vars)